xarray/backends/api.py

```diff
@@ -929,8 +929,14 @@ def open_mfdataset(
 
         If a callable, it must expect a sequence of ``attrs`` dicts and a context object
         as its only parameters.
+    drop_variables: str or iterable of str, optional
+        A variable or list of variables to exclude from being parsed from the
+        dataset. This may be useful to drop variables with problems or
+        inconsistent values.
```

Comment:
So this wasn't quite what I meant - generally we wouldn't add this entry here because it already exists in the docstring of `open_dataset`. We don't want to duplicate the description of every argument to `open_dataset` in the docstring of `open_mfdataset`.

Reply:
removed.